tests/tests_deployment/keycloak_api_utils.py

```diff
@@ -0,0 +1,944 @@
+"""Helper class for Keycloak API interactions."""
+
+import base64
+import json
+
+import requests
+
+TIMEOUT = 10
```

Comment:
Should this be configurable with a default? I am generally leery of hardcoded timeouts

Reply:
fixed